db/migrate/20150329211903_create_compliments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateCompliments < ActiveRecord::Migration
+  def change
+    create_table :compliments do |t|
+      t.string :message, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.